NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -355,10 +355,10 @@ public class NVActivityIndicatorView: UIView {
      
      - returns: The activity indicator view.
      */
-    public init(frame: CGRect, type: NVActivityIndicatorType = DEFAULT_TYPE, color: UIColor = DEFAULT_COLOR, padding: CGFloat = DEFAULT_PADDING) {
-        self.type = type
-        self.color = color
-        self.padding = padding
+    public init(frame: CGRect, type: NVActivityIndicatorType? = nil, color: UIColor? = nil, padding: CGFloat? = nil) {
+        self.type = type ?? NVActivityIndicatorView.DEFAULT_TYPE
+        self.color = color ?? NVActivityIndicatorView.DEFAULT_COLOR
+        self.padding = padding ?? NVActivityIndicatorView.DEFAULT_PADDING
```

Comment:
I made this nil-able so I can pass-thru nil parameters from other functions but still retain default values. This was opposed to making the default constants public instead of private, which I didn't want to do to avoid cluttering the public API.